### PR TITLE
8263856: Github Actions for macos/aarch64 cross-build

### PR DESCRIPTION
**Pull request by AntonKozlov**
Please review a change that adds cross-compiled macos/aarch64 platform build into Github Actions test workflow.

The steps are basically copy-paste of macos/x86-64 build with the necessary platform name adjustments. Xcode 12.4 is also used for as macos/aarch64 build.

I've checked the produced build (can be found in https://github.com/AntonKozlov/jdk/actions/runs/667527036), it starts on the actual aarch64 machine.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8263856](https://bugs.openjdk.java.net/browse/JDK-8263856): Github Actions for macos/aarch64 cross-build

### Reviewers
 * [Erik Joelsson](https://openjdk.java.net/census#erikj) (@erikj79 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3270/head:pull/3270` \
`$ git checkout pull/3270`

Update a local copy of the PR: \
`$ git checkout pull/3270` \
`$ git pull https://git.openjdk.java.net/jdk pull/3270/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3270`

View PR using the GUI difftool: \
`$ git pr show -t 3270`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3270.diff">https://git.openjdk.java.net/jdk/pull/3270.diff</a>

</details>

### CODE REVIEW

File: .github/workflows/submit.yml [hunk 1]
```diff
@@ -1362,6 +1364,110 @@ jobs:
...
+
+      - name: Install dependencies
+        run: brew install make
+
+      - name: Configure
+        run: >
+          bash configure
+          --with-conf-name=macos-aarch64
+          --openjdk-target=aarch64-apple-darwin
+          --with-extra-cflags="-arch arm64"
```
Comment (erikj79) [inline .github/workflows/submit.yml:1444]: If configure isn't adding these flags automatically, that's something that we should fix.

**Review by erikj79 — COMMENTED** [review]
Even if what you currently want is the "latest" Xcode, I still think the choice should be explicit on the version. Otherwise we will suddenly get a new version picked at some random time in the future, which may not work. It's very important to control as many external variables as possible when defining a verification build like this.

File: .github/workflows/submit.yml (hunk 1 above)
Comment (AntonKozlov) [reply .github/workflows/submit.yml]: Thanks for review! The problem with flags is resolved by merging in master with https://bugs.openjdk.java.net/browse/JDK-8264650

The Xcode version if fixed to 12.4, which is consistent with the current state of PR #3258 (https://bugs.openjdk.java.net/browse/JDK-8264224)

**Review by erikj79 — APPROVED** [review]

**Pull request closed without merging**